### PR TITLE
Fix dependency URI path for downstream projects + Karaf feature verification

### PR DESCRIPTION
**Pull request by tbouron**

### CODE REVIEW

**Comment by tbouron** [comment]
For some reason, the official maven karaf plugin cannot verify this feature if the quotes are not single quotes AND properly URI encoded, hence the `%27`

File: karaf/features/src/main/feature/feature.xml [hunk 1]
```diff
@@ -256,7 +256,7 @@
             https://github.com/square/okhttp/pull/2246
         -->
         <!-- TODO use ServiceMix bundle for okhttp when dependency on okio is fixed there -->
-        <bundle dependency="true">wrap:mvn:com.squareup.okhttp/okhttp/2.2.0$Import-Package=org.apache.http.*;resolution:=optional,android.util.*;resolution:=optional,okio;version="[1.2.0, 1.3.0)",*</bundle>
+        <bundle dependency="true">wrap:mvn:com.squareup.okhttp/okhttp/2.2.0$Import-Package=org.apache.http.*;resolution:=optional,android.util.*;resolution:=optional,okio;version=%27[1.2.0,1.3.0)%27,*</bundle>
```
Comment (aledsage) [inline karaf/features/src/main/feature/feature.xml]: @tbouron Why did you go for single quotes rather than double quotes (`%22`), out of interest?

**Comment by aledsage** [comment]
I understand that this has been confirmed manually with a mvn build, checking in karaf that okhttp bundle was added correctly. Therefore merging now.

File: karaf/features/src/main/feature/feature.xml (hunk 1 above)
Comment (tbouron) [inline karaf/features/src/main/feature/feature.xml]: Tested with encoded double quotes (`%22`) but the feature verification was still failing. Not sure why unfortunately

**Pull request merged**